src/main/java/org/helpberkeley/memberdata/UserExporter.java

```diff
@@ -63,16 +62,19 @@ String errorsToFile(final String fileName) throws IOException {
         return outputFileName;
     }
 
-    String consumerRequests() {
+    String consumerRequests() throws IOException {
+        try (StringWriter writer = new StringWriter()) {
```

Comment:
We can make all of these exposed IOExceptions in the Exporter (User, Workflow, Driver) classes go away by added a catch IOException block to the try-with-resources.